index.html

```diff
@@ -0,0 +1,115 @@
+            <div class="logo">
+                <img src="project-image/logo.jpg" alt="image">
+            </div>
+            <ul class="desktop-menu">
+                <li class="nav-item">
+                    <a href="about.html" class="nav-link active">About</a>
+                </li>
+                <li class="nav-item">
+                    <a href="#program" class="nav-link">Services</a>
+                </li>
+                <li class="nav-item">
+                    <a href="#join" class="nav-link">Instructors</a>
+                </li>
+                <li class="nav-item">
+                    <a href="#sponsor" class="nav-link">Sponsor</a>
+                </li>
+                <li class="nav-item">
+                    <a href="#news" class="nav-link">News</a>
+                </li>
+                <li class="nav-item">
+                    <p class="campaign">Apply Programs</p>
+                </li>
+            </ul>
+            
+            <div class="mobile-menu">
+                <span class="bar"></span>
+                <span class="bar"></span>
+                <span class="bar"></span>
+            </div>
+        </nav>
+    </header>
+    
+    <main>
+        <section class="container1">
+            <section class="intro">
+                <p class="hello">"Hello! OPen to work "</p>
+                <h1 class="creative">Gateway is open for software development and training.  </h1>
+            </section>
+    
+            <section class="joy">
+                <p>We offer different services such as constructing, altering, repairing, or modifying garments for customers based on their specifications, needs, and preferences</p>
+                <p>Also we take our customers' measurements, assist in faric selection, and arrange fittings to determine whether additional adjustments are needed.</p>
+            </section>
+            <section class="date-place">
+                <p class="date">2023 (Fri) ~ (Sun)</p>
+                <p class="place"> @ Ethiopia,Addis ababa,megenagna,No.office:312 </p>
+            </section>
+        </section>
+        
+        <section class="main-prog-contents" id="program">
+            <h2 class="main-prog">Main Services</h2>
+            <div class="h-r1"></div>
+            <ul class="program-list"></ul>
+            <button type="button" class="join-btn">see our product here</button>
+            <a href="#" class="all-project">Apply for program</a>
+        </section>
+            
+        <section class="speakers-section" id="join">
+            <h3 class="feature">Leading  Instructors</h3>
+            <div class="h-r2"></div>
+            <ul class="speakers-list"></ul>
+            <div class="toggle-contents">
+                <ul class="speakers-list2"></ul>
+                <ul class="speakers-list3"></ul>
+            </div>
+            <button id="toggle-btn">more &nbsp; &nbsp;<i class="fa-solid fa-angle-down angle"></i></button>
+        </section>
```

Comment:
- Kindly, MInimize the space between the speaker to stick the given design and have better Ui 👍 
#### your Design 👇
<img width="904" alt="2023-08-10_11-54" src="https://github.com/abel12-pro/Capstone-1/assets/52010789/d44286c5-7934-4717-853f-b1025dbc6382">

#### Your Design 👇
<img width="668" alt="2023-08-10_11-53" src="https://github.com/abel12-pro/Capstone-1/assets/52010789/d2378a3c-1849-417e-81ed-821095733379">